rotkehlchen/tests/unit/decoders/test_velodrome.py

```diff
@@ -380,15 +388,17 @@ def test_remove_liquidity_v1(optimism_transaction_decoder, optimism_accounts):
 
 
 @pytest.mark.vcr()
+@pytest.mark.parametrize('load_global_caches', [['velodrome']])
 @pytest.mark.parametrize('optimism_accounts', [['0x78C13393Aee675DD7ED07ce992210750D1F5dB88']])
-def test_swap_eth_to_token_v2(optimism_accounts, optimism_transaction_decoder):
+def test_swap_eth_to_token_v2(optimism_accounts, optimism_transaction_decoder, load_global_caches):
```

Comment:
here for example we do as I said. Providing `optimism_transaction_decoder` as argument we wouldn't need to patch later neither build again the object. Here we are building it twice